webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,608 @@
+      "ECE",
+    ].flatMap((field) => [
+      [`comparedPipeline${field}`, visibility],
+      [`delta${field}`, visibility],
+    ]);
+    setColumnVisibilityModel(Object.fromEntries(columnVisibilityEntries));
+  }, [pipeline, comparedPipeline, metricInfo]);
+
+  const RowLink = (props: RowProps<Row>) => (
+    <Link
+      style={{ color: "unset", textDecoration: "unset" }}
+      to={`/${jobId}/dataset_splits/${selectedDatasetSplit}/performance_overview${constructSearchString(
+        {
+          ...(props.row.id !== OVERALL_ROW_ID && {
+            [selectedMetricPerFilterOption]: [
+              props.row.basePipeline.filterValue,
+            ],
+          }),
+          ...pipeline,
+        }
+      )}`}
+    >
+      <GridRow {...props} />
+    </Link>
+  );
+
+  return errorFetchingBasePipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingBasePipelineData.message}
+    </Typography>
+  ) : errorFetchingComparedPipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingComparedPipelineData.message}
+    </Typography>
+  ) : (
+    <Paper
+      variant="outlined"
+      sx={{
+        height: "100%",
+        padding: 4,
+        marginTop: 2,
+      }}
+    >
+      <Box marginBottom={1} display="flex" justifyContent="space-between">
+        <Box width={340}>
+          <DatasetSplitToggler
+            availableDatasetSplits={availableDatasetSplits}
+            value={selectedDatasetSplit}
+            onChange={setSelectedDatasetSplit}
+          />
+        </Box>
+        <FormControlLabel
+          label={`Compare Baseline (${
+            config?.pipelines?.[pipeline.pipelineIndex].name
+          } ) with:`}
+          labelPlacement="start"
+          sx={{ gap: 1, paddingRight: 2 }}
+          control={
+            <Select
+              id="compare-pipeline-model-select"
+              variant="standard"
+              value={comparedPipeline ?? "No pipeline"}
+              displayEmpty
+              onChange={({ target: { value } }) =>
+                setComparedPipeline(
+                  typeof value === "number" ? value : undefined
+                )
+              }
+            >
+              <MenuItem value="No pipeline">
+                <em>No pipeline</em>
+              </MenuItem>
+              {config?.pipelines?.map(
+                (pipelineData, index) =>
+                  index !== pipeline.pipelineIndex && (
+                    <MenuItem key={index} value={index}>
+                      {pipelineData.name}
+                    </MenuItem>
+                  )
+              )}
+            </Select>
+          }
+        ></FormControlLabel>
+      </Box>
+      <Paper
+        variant="outlined"
+        sx={{
+          marginTop: 2,
+          padding: 3,
+        }}
+      >
```

Comment:
Any reason for the outlined paper here? That's not the look we are going for in the rest of the app. In fact, we specifically remove the border of the `Table` here:
https://github.com/ServiceNow/azimuth/blob/d3c1e6b3647adf5bccb2e89565c3182d1ad99ec3/webapp/src/components/Table.tsx#L96
Can you change that for a `Box`?
```suggestion
      <Box marginTop={2} padding={3}>
```

Reply:
Yes, right, we don't have borders in other places. I added this because of the design here https://www.figma.com/proto/FEndxTlShYTSUjMrInKqqn/ModelExplainer-V1?page-id=518%3A12557&node-id=1833%3A32699&viewport=1484%2C9621%2C0.5&scaling=min-zoom&starting-point-node-id=1833%3A32699.

Reply:
> 

Okay, so I am just curious to learn here, so is it good to use a box instead paper mui?

Reply:
> 

Understood. borders can be consistent like all other tables. it's for the design I added but not really necessary. what do you think?

Reply:
Yes, I think `Box` is better here, as `Paper` is meant to be the background of the app, like the sheet of paper we draw the app on.

Reply:
I'm sorry, I was confused here by the two nested `Paper`s. I mean to comment on the second one. I'll write new comments...